core/utils/cuckoo_map.h

```diff
@@ -206,6 +206,73 @@ class CuckooMap {
     return entry;
   }
 
+  // Insert/update a key value pair
+  // Return the pointer to the inserted entry
+  Entry *Insert(const K &key, V &&value, const H &hasher = H(),
+                const E &eq = E()) {
+    Entry *entry;
```

Comment:
Just for clarifying,
In comments, you said that 'move Insert()', but here you create another Insert with Rvalue-Reference. Which one do you intended?

Reply:
Ah, my intention was the latter. I'll update the commit message to be clearer

Reply:
Got it